src/components/PlanListPage.jsx

```
@@ -0,0 +1,157 @@
+                key={tab}
+                className={tab === activeTab ? 'tab active' : 'tab'}
+                onClick={() => setActiveTab(tab)}
+              >
+                {tab}
+              </button>
+            )
+          )}
+        </nav>
+
+        {/* 우측 로그인 */}
+        <button className="login-btn" onClick={() => navigate('/')}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)